include/silo/storage/column/pango_lineage_column.h

```diff
@@ -7,22 +7,27 @@
 #include <roaring/roaring.hh>
 
 #include "silo/common/pango_lineage.h"
+#include "silo/storage/column/column.h"
 
 namespace silo::storage::column {
 
-class PangoLineageColumn {
+class PangoLineageColumn : public Column {
   public:
    template <class Archive>
    [[maybe_unused]] void serialize(Archive& archive, const unsigned int /* version */) {
+      archive& values;
       archive& value_id_lookup;
+      archive& id_value_lookup;
       archive& indexed_values;
+      archive& indexed_sublineage_values;
    }
 
   private:
+   std::vector<uint32_t> values;
```

Comment:
```suggestion
   std::vector<uint32_t> value_ids;
```
Wouldn't that be more correct?